### ISSUE TITLE
Are there tissue samples available? 

### ISSUE THREAD

**Issue by jackwgoodall** [GoekeLab/sg-nex-data#66]
Thank you for this amazing resource. On the AWS landing page it sounds like there are tissue samples processed but I only seem able to find cell lines sequenced?  
Apologies if this is just me missunderstanding either the data description or the file descriptions.

**Comment by cying111** [comment]
Hi @jackwgoodall , 

thank you for using SG-NEx data resource and for pointing out that the description is unclear.  For the core datasets, we have generated 7 core cell lines, and we have also sequenced patient-derived cell lines (which we refer as tissue samples). Then for core cell lines, we have sequenced them in multiple replicates using different sequencing protocols. For patient-derived cell lines, however, we sequenced with direct RNA only and not in replicates. For your information, here are the patient-derived cell lines:  NCC24, IM95 and N082, N122, and N104.

Hope this clarifies your questions and let me know if you have other questions!
Thank you
Warm regards,
Ying
